Products/CMFPlone/controlpanel/browser/configure.zcml

```diff
@@ -111,6 +111,35 @@
       permission="cmf.ManagePortal"
       />
 
+  <!-- Prefs Error Log Form Control Panel -->
+  <browser:page
+      name="prefs_error_log_form"
+      for="Products.CMFPlone.interfaces.IPloneSiteRoot"
+      class=".prefs_error_log_form.PrefsErrorLogForm"
```

Comment:
Since the class does not really have any code, you can remove it here, and use `template="prefs_error_log_form.pt"`.